libs/libpugiutil/src/pugixml_loc.cpp

```diff
@@ -30,7 +30,7 @@ void loc_data::build_loc_data() {
 
     std::ptrdiff_t offset = 0;
 
-    char buffer[1024];
```

Comment:
Use same defined constant (put in some header, with a comment, or in as a -D make/cmake variable with a comment)